siliconcompiler/core.py

```diff
@@ -415,6 +416,13 @@ def load_target(self, module, **kwargs):
             Loads the 'freepdk45_demo' target with 5 parallel synthesis tasks
         """
 
+        self.set('dependency', 'siliconcompiler_data',
+                 'path', 'git+https://github.com/siliconcompiler/siliconcompiler')
+        self.set('dependency', 'siliconcompiler_data',
+                 'name', 'siliconcompiler')
+        self.set('dependency', 'siliconcompiler_data',
+                 'commitid', 'dependency-caching-rebase')
+
```

Comment:
I think we should set this or something like it in the `__init__` method

Reply:
I don't think that makes much sense as this is only required for targets.
Otherwise the tests right fully complain that we have set a default that does not correspond to the schema default.
We could set the siliconcompiler as a default in the schema which I tried but couldn't get to work. But I can look more into that.

Reply:
this feels a little hacky here, lets leave it and figure it out before merging